XenModel/XenAPI/VGPU.cs

```diff
@@ -50,6 +50,13 @@ public partial class VGPU : XenObject<VGPU>
         public VGPU()
         {
         }
+        /// <summary>

```

Comment:
All files in `XenModel/XenAPI` come from the SDK and are autogenerated. No edits should be done in XenCenter because they will be overwritten next time we update the bindings.

Reply:
We don't really need this constructor. We can assign the device after the object is created, as we do with other properties.